shielddash/api/models.py

```diff
@@ -1,8 +1,36 @@
 from django.db import models
+from django.utils import timezone
 
 
-class Retention(models.Model):
+class Study(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(default='')
```

Comment:
It would make the dashboard nice to explain the goal of the study but I don't remember seeing that in Normandy.